papaya-core/src/main/java/pl/edu/agh/papaya/service/project/ProjectCreationWizard.java

```diff
@@ -31,11 +34,25 @@ public ProjectCreationWizard withInitialCoefficient(double initialCoefficient) {
         return this;
     }
 
+    public ProjectCreationWizard withWebhook(String webhookUrl) {
+        this.webhookUrl = webhookUrl;
+        return this;
+    }
+
+    public ProjectCreationWizard withChannelName(String channelName) {
+        this.channelName = channelName;
+        return this;
+    }
+
     public Project create() {
         Project project = new Project();
         project.setName(require("name", name));
         project.setDescription(require("description", description));
         project.setInitialCoefficient(require("initialCoefficient", initialCoefficient));
+        if (new UrlValidator().isValid(webhookUrl)) {
```

Comment:
Tu jest dla ułatwienia na froncie założenie, że jak adres webhooka nie jest poprawny, to po cichu ignorujemy ten fakt i udajemy, że go nie było w ogóle